### ISSUE TITLE
There should probably be a ICal.rakumod

### ISSUE THREAD

**Issue by JJ** [Xliff/p6-ICal#7]
Even if it's only for reference. Could really help navigate the rest of the files.

**Comment by Xliff** [comment]
Yes, but it will be a while before I standardize on .rakumod over .pm6.

**Comment by JJ** [comment]
.pm6 is OK too. Even .pod6, if all you want is a placeholder with
documentation.

**Comment by Xliff** [comment]
@JJ - I will think about that. Might be a decent idea. What are the protocols for grabbing a namespace? ICal.pm6 would basically mean that I'm taking responsibility for ICal::*

**Comment by JJ** [comment]
It's land grab, as in Oklahoma. Go as far as your repos reach in a single day, and all that.

**Comment by JJ** [comment]
And, no, you're not taking responsibility for the namespace; `zef` separates by module, so you could have X and you would no be in charge of the whole Exception conventional namespace.
